src/openms/source/METADATA/DataProcessing.cpp

```diff
@@ -75,58 +75,57 @@ namespace OpenMS
   {
   }
 
-  bool DataProcessing::operator==(const DataProcessing & rhs) const
+  bool DataProcessing::operator==(const DataProcessing& rhs) const
   {
     return software_ == rhs.software_ &&
```

Comment:
Is there something that might be faster to compare first? e.g., the completion_time_?

Reply:
Premature optimisation? In any case, not my code.